===================================================================
RCS file: /cvs/gnome/gnome-control-center/gnome-settings-daemon/gnome-settings-multimedia-keys.c,v
retrieving revision 1.6
retrieving revision 1.10
diff -u -r1.6 -r1.10
--- gnome-settings-multimedia-keys.c	2004/03/24 02:54:32	1.6
+++ gnome-settings-daemon/gnome-settings-multimedia-keys.c	2004/11/30 12:02:18	1.10
@@ -645,17 +651,7 @@
 static void
 do_exit_action (Acme *acme)
 {
-	GnomeClient *master;
-
-	master = gnome_master_client();
-	g_return_if_fail(master != NULL);
-
-	gnome_client_request_save(master,
-			GNOME_SAVE_BOTH,
-			TRUE,
-			GNOME_INTERACT_ANY,
-			FALSE,
-			TRUE);
+	execute ("gnome-session-save --kill", FALSE);
 }
 
 static void
